### PR TITLE
gpg version fixes

### PR DESCRIPTION
**Pull request by otakup0pe**
Also updates test data until I get around to #17

### CODE REVIEW

File: cryptorito/__init__.py [hunk 1]
```diff
@@ -176,14 +176,26 @@ def gnupg_verbose():
...
-    """Return the path to the gpg binary"""
-    cmd = ["which", "gpg2"]
-    output = stderr_output(cmd).strip().split('\n')[0]
-    if not output:
-        raise CryptoritoError("gpg2 must be installed")
+    """Return the path to the gpg binary.
+    Note that on some systems this is gpg, and others gpg2. We
+    try to support both."""
+    gpg_output = which_bin("gpg2")
+    if not gpg_output:
```
Comment (aaronaddleman) [inline cryptorito/__init__.py]: maybe consider a while loop over array of items to look for... i dislike nested ifs, but its a pure personal preference

File: cryptorito/__init__.py [hunk 2]
```diff
@@ -208,14 +220,16 @@ def fingerprint_from_keybase(fingerprint, kb_obj):
         for key in kb_obj['public_keys']['pgp_public_keys']:
             keyprint = fingerprint_from_var(key).lower()
             fingerprint = fingerprint.lower()
-            print("ASDASD  %s %s", [keyprint, fingerprint])
             if fingerprint == keyprint or \
-               keyprint.startswith(fingerprint):
+               keyprint.startswith(fingerprint) or \
+               keyprint.endswith(fingerprint):
```
Comment (aaronaddleman) [inline cryptorito/__init__.py:224]: awesome

File: cryptorito/__init__.py [hunk 3]
```diff
@@ -349,7 +363,7 @@ def import_gpg_key(key):
 def export_gpg_key(key):
     """Exports a GPG key and returns it"""
     cmd = flatten([gnupg_bin(), gnupg_verbose(), gnupg_home(),
-                   "--export", key, "--armor"])
```
Comment (aaronaddleman) [inline cryptorito/__init__.py:352]: why?

 -a, --armor                   create ascii armored output

ascii seems like a good thing

File: cryptorito/__init__.py (hunk 2 above)
Comment (otakup0pe) [reply cryptorito/__init__.py:224]: yeah apparently some tools use the first 8, some the last 8....

**Review by aaronaddleman — APPROVED** [review]
I love it!

File: cryptorito/__init__.py (hunk 3 above)
Comment (otakup0pe) [reply cryptorito/__init__.py:352]: [propriecle](https://github.com/autodesk/propriecle) needs base64. Doing this and then base64'ing manually produces more consistent behavior across gpg versions.

**Pull request merged**